### DISCUSSION TITLE
v0.6

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#299] [New Releases]
## What's Changed

* Add ability to restore opened and unsaved files between application restarts
* Switch to Qt6 for official distribution (Flatpak distribution will remain on v5.15)
* Build universal (multi-architecture) binaries for MacOS for native ARM support
* Add Column Mode dialog to insert text/numbers at a specific column
* Add File List to show opened editors
* Add copying/exporting as HTML and RTF
* Add app icon to Linux AppImage
* Open all files when folder is dropped into the application
* Allow renaming of temp files
* Allow editing characters through the Hex Viewer
* Update Lexilla to v5.2.1
* Update Scintilla to v5.3.2
* Default to UTF-8 for empty files
* Fix crash on Linux when secondary instance opens non-existent file
* Fix replacing all matches in a document

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.5.6...v0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.6'>v0.6</a>.</em>

**Comment by jialu0** [comment]
How do I change the UI language from Chinese to English?

**Reply by trdm** [reply]
> Switch to Qt6 for official distribution (Flatpak distribution will remain on v5.15)

;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;( ;(

**Reply by dail8859** [reply]
@jialu0 It tries to default to your system locale, that's why it is loading chinese. Currently the only way to override the system detected language is to try the `--translation` command line flag. Eg

```
notepadnext.exe --translation en_us
```

**Reply by dail8859** [reply]
@trdm There are upstream issues outside of my control. https://github.com/flathub/com.github.dail8859.NotepadNext/pull/12

**Reply by trdm** [reply]
> @trdm There are upstream issues outside of my control. [flathub/com.github.dail8859.NotepadNext#12](https://github.com/flathub/com.github.dail8859.NotepadNext/pull/12)

Тогда прощай NotepadNext....
остаюсь с Notepad+ + windosw7.
PS. I don`t can understand you, my English still low level.
